src/main/java/org/sagebionetworks/web/client/view/ProfileViewImpl.java

```diff
@@ -516,22 +545,59 @@ public void setTabSelected(Synapse.ProfileArea targetTab) {
 
     if (targetTab == Synapse.ProfileArea.PROFILE) {
       pageHeaderTitle.setText(profileHeaderText);
+      orientationBannerPanel.clear();
       DisplayUtils.show(profileTabContainer);
     } else if (targetTab == Synapse.ProfileArea.PROJECTS) {
       pageHeaderTitle.setText("Your Projects");
+      configureOrientationBanner(
+        "Projects",
+        "Getting Started With Your Projects",
+        "Projects are the main \"containers\" where information is stored and organized in Synapse. They are online workspaces where you can collaborate and share your work with teammates. Projects can be shared with individuals, small teams, or large consortia. Projects can be private so only you and your team can see what\'s inside, or they can be shared publicly for anyone to browse.",
+        null,
+        null,
+        "Learn More About Projects",
+        "https://help.synapse.org/docs/Setting-Up-a-Project.2055471258.html"
+      );
       DisplayUtils.show(projectsTabContainer);
     } else if (targetTab == Synapse.ProfileArea.TEAMS) {
       pageHeaderTitle.setText("Your Teams");
+      configureOrientationBanner(
+        "Teams",
+        "Getting Started With Your Teams",
+        "Teams allow you to easily manage groups of users to control access to projects, communicate with colleagues, and participate in challenges.",
+        "Search Teams",
+        "https://www.synapse.org/#!TeamSearch:",
```

Comment:
Link takes you to the wrong stack if you're not on prod--not a huge deal but would like to fix. Is it possible to use a callback here instead of an href and use `goToPlace`?